content/docs/command-reference/remote/modify.md

```diff
@@ -315,19 +315,103 @@ $ dvc remote modify myremote endpointurl \
   $ dvc remote modify myremote url azure://mycontainer/path
   ```
 
-- `connection_string` - connection string:
+#### Authentication Flow:
+
+- If `connection_string` specified, it will auth with it
+- If `tenant_id`/`client_id`/`client_secret` is specified, it will auth with AD
+  service principal
+- If `account_name`/`account_key` is specified, it will auth with account_key
+- If `account_name`/`sas_token` is specified, it will auth with SAS token
+- If `account_name` is specified, it will auth anonymously
```

Comment:
@isidentical How can it be "anonymous" if an account **name** is used? (Last bullet) Thanks

Reply:
Are we talking about this (https://docs.microsoft.com/en-us/azure/storage/blobs/anonymous-read-access-configure?tabs=portal) and if so does it depend on the storage config?

Reply:
> @isidentical How can it be "anonymous" if an account name is used? (Last bullet) Thanks

It is the location to your storage service. So if you specify your account name only (e.g `test_iterative`) and then add `azure://temp-bucket/data` as a remote then it will access `test_iterative.azuresomething.com/test-bucket/data` under the hood anonymously (unless you give any of the authentication options with it).

Reply:
I see, thanks.

So yes to the link and Q I mentioned? I.e. it depends on the storage access being setup public.

Reply:
> It is the location to your storage service

Also, a bit confusing to call it "account name" if it's a path or location. What is the same concept called in Azure, is it "storage account"? If so maybe this param should be `storage_account`.